auth_oauth_multi_token/tests/test_multi_token.py

```diff
@@ -4,28 +4,24 @@
 import json
 
 from odoo import exceptions
-from odoo.tests import SavepointCase
+from odoo.tests import TransactionCase, tagged
 
 
-class TestMultiToken(SavepointCase):
-
-    post_install = True
-    at_install = False
-
-    @classmethod
-    def setUpClass(cls):
-        super().setUpClass()
-        cls.token_model = cls.env["auth.oauth.multi.token"]
-        cls.provider_google = cls.env.ref("auth_oauth.provider_google")
-        cls.user_model = cls.env["res.users"].with_context(
-            {"tracking_disable": True, "no_reset_password": True}
+@tagged("-at_install", "post_install")
```

Comment:
Why you need post-install?

Reply:
What about this? If you need post_install by any reason, please put a comment explaining why for not being removed in later revisions.